plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -355,7 +373,9 @@ private void deleteModelChunksAndController(
             // for remote model we don't need to delete model chunks so reducing one latch countdown.
             countDownLatch.countDown();
         }
-        deleteController(modelId, isHidden, countDownActionListener);
+        // TODO This uses remote client but we haven't implemented it anywhere else?!
+        // deleteController(modelId, isHidden, countDownActionListener);
```

Comment:
I assume you will reverse the code for controller deletion and then enable this, right?

Reply:
> I assume you will reverse the code for controller deletion and then enable this, right?

Yep, as mentioned in the original comment.

I'll put it back to its pre #2585 state.  If there are any workarounds still needed, I understand a permanent fix is coming with feature-gating in #2652

Reply:
Except that commenting it out is causing tests to fail.  Let me pull it back out of this PR and address separately.